integration/podman/run_podman_tests.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../../.ci/lib.sh"
+source /etc/os-release || source /usr/lib/os-release
+
+podman_repository="github.com/containers/libpod"
+podman_repository_path="${GOPATH}/src/${podman_repository}"
+# Podman configuration file
+export PODMAN_CONFIG="${SCRIPT_PATH}/../../.ci/podman/configuration_podman.yaml"
+# filter scheme script for podman integration test suites
+export PODMAN_FILE="${SCRIPT_PATH}/../../.ci/filter/filter_podman_test.sh"
+test_repository="github.com/kata-containers/tests"
+test_repository_path="${GOPATH}/src/${test_repository}"
+version=$(get_test_version "externals.podman.version")
+
+function setup() {
+	# Clone podman repository if it is not already present.
+	if [ ! -d "${podman_repository_path}" ]; then
+		go get -d "${podman_repository}" || true
+	fi
+
+	# Clone test repository if it is not already present
+	if [ ! -d "${test_repository_path}" ]; then
```

Comment:
do we need this? If we are using this script is because test repository is already cloned, right?